lupa/tests/test.py

```diff
@@ -3007,10 +3007,19 @@ def setUp(self):
             return self.skipTest("not supported in LuaJIT")
         return super(TestMaxMemory, self).setUp()
 
-    def test_property(self):
-        self.assertEqual(self.lua.max_memory, 10000)
+    def test_getters(self):
+        self.assertEqual(self.lua.get_memory_used(), 0)
+        self.assertGreater(self.lua.get_memory_used(True), 0)
```

Comment:
Please use keyword arguments instead of bare truth values to help readers understand what this does without looking up the function signature.